frontend/workflows/projectSelector/src/project-links.tsx

```diff
@@ -178,26 +189,32 @@ const QuickLinksPopper = ({
 
 interface ProjectLinksProps {
   linkGroups: IClutch.core.project.v1.ILinkGroup[];
-  name: string[];
+  setKeyWithQLinksOpen: (key: string) => void;
+  keyWithQLinksOpen: string;
+  currentKey: string[];
```

Comment:
why is this a list?

Reply:
daniel's suggestion was he wanted it passed as a list - I'm not really sure why but that's what he said, to pass it as `[key]` not as `key`. https://github.com/lyft/clutch/pull/2191#discussion_r855345873 Or maybe I just misinterpreted his comment.